src/box/memtx_engine.cc

```diff
@@ -66,6 +66,27 @@
 
 #include <type_traits>
 
+static inline uint32_t
+mh_ptr_hash(const void *ptr)
+{
+	uintptr_t u = (uintptr_t)ptr;
+	return u ^ (u >> 32);
+}
+
+/**
+ * Set: (void *)
+ */
+#define mh_name _ptr
```

Comment:
I guess this could be useful somewhere else. Let's put it in `assoc.h` (in a separate commit).

Reply:
Done.